CityVitalsWatchSettings.cs

```diff
@@ -102,6 +102,11 @@ public class CityVitalsWatchSettings {
         /// </summary>
         public bool DisplayEmployment = true;
 
+        /// <summary>
+        /// Indicates whether job availability should be displayed.
+        /// </summary>
+        public bool DisplayJobAvailability = true;
```

Comment:
New meter visibility should be false by default, as we don't want to edit users' current layout without them knowing.

Reply:
Hi Rob - makes sense, I've updated it.

Thanks again for creating this mod - it's been on my list of essentials for years.  I'm glad I can contribute!